client/api/routes.js

```diff
@@ -0,0 +1,65 @@
+const express = require('express');
+const router = express.Router();
+const { healthCheck, updateSignText } = require('../ledsign/led_sign_client');
+const { sendPrintRequest } = require('../printing/print_client');
+const { send3dPrintRequest } = require('../printing_3d/print_3d_client');
+const {
+  OK, BAD_REQUEST, NOT_FOUND, LED_SIGN_IP
+} = require('../config/config');
+const { addSignLog } = require('../util/logging-helpers');
+
+router.post('/LedSign/healthCheck', (req, res) => {
+  const { officerName } = req.body;
+  healthCheck(officerName, LED_SIGN_IP)
+    .then(response => {
+      const { message } = response;
+      return res.status(OK).send({
+        text: message.getText(),
+        brightness: message.getBrightness(),
+        scrollSpeed: message.getScrollSpeed(),
+        backgroundColor: message.getBackgroundColor(),
+        textColor: message.getTextColor(),
+        borderColor: message.getBorderColor()
+      });
+    })
+    .catch(error => {
+      return res.status(NOT_FOUND).send({ ...error });
+    });
+});
+
+router.post('/LedSign/updateSignText', async (req, res) => {
+  if (!await addSignLog(req.body)) {
+    return res.sendStatus(BAD_REQUEST);
+  }
+  updateSignText(req.body, LED_SIGN_IP)
+    .then(response => {
+      return res.status(OK).send({ ...response });
+    })
+    .catch(error => {
+      return res.status(NOT_FOUND).send({ ...error });
+    });
+});
+
+router.post('/Printer/sendPrintRequest', (req, res) => {
+  const { raw, copies, sides, pageRanges, destination } = req.body;
+  sendPrintRequest(raw, copies, sides, pageRanges, destination)
+    .then(response => {
+      return res.status(OK).send({ ...response });
+    })
+    .catch(error => {
+      return res.status(BAD_REQUEST).send({ ...error });
+    });
+});
+
+router.post('/3dPrinter/print3dModel', (req, res) => {
+  const { raw, memberName, volume, copies } = req.body;
+  send3dPrintRequest(raw, memberName, volume, copies)
+    .then(response => {
+      return res.status(OK).send({ ...response });
+    })
+    .catch(error => {
+      return res.status(BAD_REQUEST).send({ ...error });
+    });
+});
+
+module.exports = router;
```

Comment:
Why did you decide to put all the routes in one file? Are you not afraid of this file getting super long if we add more RPCs or more functionality to existing ones?

Reply:
i didnt wanna have a big amount of files start accumulating in the API directory. i think this is a good idea though, should i split it up?

Reply:
I think it would be a good idea to. Since a lot of the intern RPC projects will add new routes as well. Finding the route you're having problems with might become harder in the future.

Reply:
Factual, let me split it up now

Reply:
@law-nguyen should be done now.